.github/workflows/administration-check.yaml

```diff
@@ -1,43 +1,59 @@
-name: 'Administration Check'
+name: 'PR'
+
 on:
   pull_request:
     branches:
       - main
     paths:
-      - 'administration/**'
+      - .prettierrc
+      - eslint.config-base.mjs
+      - graphql.config.yml
+      - package.json
+      - package-lock.json
+      - administration/**
+      - frontend/build-configs/**
+
 jobs:
   administration-check:
+    name: 'administration check'
     runs-on: ubuntu-latest
+
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v4
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup node
+        uses: actions/setup-node@v6
         with:
           node-version: '20.18.0'
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
-      - name: Cache node modules
-        uses: actions/cache@v4
-        with:
-          path: ~/.npm
-          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-node-
+
       - name: Install node modules
-        run: npm ci --cache ~/.npm --prefer-offline
-      - name: Typescript
+        run: npm ci
+
+      - name: Check Typescript
         run: npm run ts:check
         working-directory: administration
-      - name: Lint
+
+      - name: Run Lint
         run: npm run lint
         working-directory: administration
+
       - name: Check circular dependencies
         run: npm run check-circular-deps
         working-directory: administration
-      - uses: ArtiomTr/jest-coverage-report-action@v2
-        name: Run tests with coverage
+
+      - name: Run tests with coverage
+        uses: ArtiomTr/jest-coverage-report-action@v2
+        if: github.event.pull_request.user.login != 'dependabot[bot]'
         with:
           base-coverage-file: report.json
           skip-step: install
           working-directory: administration
           test-script: npm run test:coverage -- --silent --ci --json --testLocationInResults --outputFile=report.json --maxWorkers=1
 
+      - name: Run tests
+        if: github.event.pull_request.user.login == 'dependabot[bot]'
+        run: npm run test
```

Comment:
❓ I wonder if the `--maxWorkers=1` parameter is only needed for coverage, but not for tests (I don't know why it was added in the first place, maybe there were some parallelism issues in the tests?).